pkg/cloud/services/autoscaling/autoscalinggroup.go

```diff
@@ -323,9 +323,6 @@ func (s *Service) CanStartASGInstanceRefresh(scope *scope.MachinePoolScope) (boo
 	describeInput := &autoscaling.DescribeInstanceRefreshesInput{AutoScalingGroupName: aws.String(scope.Name())}
 	refreshes, err := s.ASGClient.DescribeInstanceRefreshesWithContext(context.TODO(), describeInput)
 	if err != nil {
-		if awserrors.IsNotFound(err) {
```

Comment:
For the record, this is removed because AWS returns `ValidationError` instead of a clear "not found" code, right?

Reply:
Exactly. It never returns an error that would match `IsNotFound()` so it's useless to have the check here.